frontend/src/pages/dashboard/user-history/index.tsx

```diff
@@ -26,7 +26,7 @@ const UserHistory: FC<DashboardCompProps> = ({ userUid }) => {
         setLoading(true);
         const resOffer = await getOffers(userUid);
         const resTask = await getTasks(userUid);
-        const resRequest = await getRequests(userUid);
+        const resRequest = await getRequests(userUid, Status.DONE);
```

Comment:
So history only shows the completed requests? 😅 What about those that were cancelled?

Reply:
Yea could I request for a history api? Right now was using like a placeholder method with GetRequests and all that

Reply:
Okay!

Reply:
Sorry, just to clarify, 'history' should show the user's entire history, shouldn't it? 😅 If so, do we need the status? Or shall I order them by status? In particular, I will return `{ tasks: {done: [...], cancelled: [..], pending: [...], ...}, offers: {open: [...], ...}, requests: {done: [...], ...}}`.

Reply:
Can be 3 endpoints or one actually dowsnt matter haha